build/ros3d.js

```diff
@@ -791,8 +790,8 @@ ROS3D.InteractiveMarker.prototype.onServerSetPose = function(event) {
       this.position.y = pose.position.y;
       this.position.z = pose.position.z;
 
-      this.quaternion = new THREE.Quaternion(pose.orientation.x, pose.orientation.y,
-          pose.orientation.z, pose.orientation.w);
+      this.quaternion.copy(new THREE.Quaternion(pose.orientation.x, pose.orientation.y,
```

Comment:
you could probably use set() here, as you did with the other quaternions further down